.github/workflows/code-scanning.yml

```diff
@@ -0,0 +1,64 @@
+name: "code-scanning"
+
+on:
+  push:
+  pull_request:
+  # Schedule code scanning
+  #schedule:
+  #  - cron: '0 1 * * 0'
+
+env:
+  MACOS_BREW_PACKAGES: "pkg-config libomp"
+
+jobs:
+
+  codeql-cpp-build-macos:
+    # For now, run the cpp analysis on macOS only for ease of grpc installation
+    runs-on: macos-latest
+    steps:
+      - name: Checkout repository and install submodules
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Install Dependencies
+        run: brew install ${MACOS_BREW_PACKAGES}
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v1
+        with:
+          languages: cpp
+        # Autobuild for compiled languages.
```

Comment:
remove?

Reply:
Ah yes, this one needs to go away, thanks I forgot to remove it.